js/models/contact_model.js

```diff
@@ -307,6 +309,7 @@ angular.module('contactsApp')
 				return matchingProps.length > 0;
 			},
 
+			/* eslint-disable no-console */
```

Comment:
Ah, this seems to be intended. Should we hide this behind some flag?
(`debug` sounds a bit wrong, maybe `console.error`, `console.warning` or `console.info`might be better.)